idea/src/META-INF/plugin.xml

```diff
@@ -2278,6 +2278,14 @@
                      language="kotlin"
     />
 
+    <localInspection implementationClass="org.jetbrains.kotlin.idea.inspections.RedundantProgressionStepInspection"
+                     displayName="Redundant progression step"
+                     groupName="Kotlin"
```

Comment:
Please provide a group path and group name for the inspection according to https://github.com/JetBrains/kotlin/commit/a09b31b5680952c56bb3ea5bbcb7939084e9f027